Extensions/AadIssuerValidator.cs

```diff
@@ -23,29 +21,29 @@ public static class AadIssuerValidator
         /// <returns>The <c>issuer</c> if it's valid, or otherwise <c>SecurityTokenInvalidIssuerException</c> is thrown</returns>
         public static string ValidateAadIssuer(string issuer, SecurityToken securityToken, TokenValidationParameters validationParameters)
         {
-            JwtSecurityToken jwtToken = securityToken as JwtSecurityToken;
+            var jwtToken = securityToken as JwtSecurityToken;
             if (jwtToken == null)
             {
                 throw new SecurityTokenInvalidIssuerException("Expecting a JWT Token from Azure Active Directory.");
             }
 
             // Extracting the tenant ID
-            string tenantId = jwtToken.Claims.FirstOrDefault(c => c.Type == "tid")?.Value;
+            var tenantId = jwtToken.Claims.FirstOrDefault(c => c.Type == "tid")?.Value;
```

Comment:
In these cases I don't think the type is apparent and it would be a bit more readable to use string.